pkg/kubelet/userns/userns_manager.go

```diff
@@ -212,7 +236,7 @@ func (m *UsernsManager) allocateOne(pod types.UID) (firstID uint32, length uint3
 
 	klog.V(5).InfoS("new pod user namespace allocation", "podUID", pod)
 
-	firstID = uint32(firstZero * userNsLength)
+	firstID = uint32((firstZero + m.off) * userNsLength)
```

Comment:
Doesn't the bitmap that is already created with offset and len account for the offset? Isn't already taken into account in what it returns, and we are adding it again here?

Reply:
the bitmap records starting at off=0